src/common/workload/k8s.py

```diff
@@ -46,7 +46,7 @@ def read(self, path: str) -> list[str]:
             raise FileNotFoundError
 
         with self.container.pull(path) as f:
-            return f.read().split("\n")
+            return f.read()
```

Comment:
**thought**: not saying that this is what's good for this code base, but kafka & zookeeper just return an empty list, which is good enough for our uses.  I am also curious as to why you felt the need the return the str content instead of keeping the splitlines.

We could get rid of a few try excepts later by returning the empty list

Reply:
Previously the `read` would return list of lines separated by `\n`, but the `write` expected an `str`. I feel that its best for `read` and `write` be inverse of each other, such that `write(read())` would write the thing that was read.

For example, I wanted to implement a logic during container restart that the files would only be written whenever the config has changed. In this case, the pseudocode would be like the following if read() and write() are inverse of each other:

```python
existing = file.read()
new_content: str
if existing != new_content:
    file.write(new_content)
```

However, if `read` returns a list and write expects a str, we would need to do a conversion `existing = "\n".join(existing)` on the fly whenever we want to compare whats in the file vs what is expected, which I don't really like. I'd rather check for the FileNotFound exception to be more explicit at the cost of extra try-catch.

Reply:
Fair enough, this is a very valid reason